dspace-api/src/main/java/org/dspace/discovery/SolrServiceImpl.java

```diff
@@ -2305,4 +2597,64 @@ public String calculateExtremeValue(Context context, String valueField,
         }
         return null;
     }
+
+    private void addNamedResourceTypeIndex(SolrInputDocument document, String acvalue, String fvalue) {
+
+        document.addField("namedresourcetype_filter", acvalue);
+
+        String[] avalues = acvalue.split(SolrServiceImpl.AUTHORITY_SEPARATOR);
+        acvalue = avalues[0];
+
+        String avalue = avalues[1];
+        document.addField("namedresourcetype_authority", avalue);
+        document.addField("namedresourcetype_group", avalue);
+        document.addField("namedresourcetype_ac", acvalue);
+
+        Pattern pattern = Pattern.compile(REGEX);
+        Matcher matcher = pattern.matcher(acvalue);
+        if (matcher.matches()) {
+            fvalue = matcher.group(1);
+        }
+
+        document.addField("namedresourcetype_keyword", fvalue);
+    }
+
+    class IndexerThread extends Thread {
+        private boolean force;
+        private List<UUID> itemids;
+
+        public IndexerThread(List<UUID> itemids, boolean force) {
+            this.force = force;
+            this.itemids = itemids;
+        }
+
+        @Override
+        public void run() {
+            Context context = null;
+            try {
+                context = new Context();
+                context.turnOffAuthorisationSystem();
+                int idx = 1;
+                final String head = this.getName() + "#" + this.getId();
+                final int size = itemids.size();
+                for (UUID id : itemids) {
+                    try {
+                        Item item = itemService.find(context, id);
+                        indexContent(context, item, force);
+                        context.uncacheEntity(item);
+                    } catch (Exception ex) {
+                        log.error("ERROR: identifier item:" + id + " identifier thread:" + head);
+                    }
+                    System.out.println(head + ":" + (idx++) + " / " + size);
+                }
+            } catch (Exception e) {
+                e.printStackTrace();
```

Comment:
Can this be removed, as it is already logged below.

Reply:
the code doesn't exist anymore